shell/probe/raindrop.ps1

```diff
@@ -4,9 +4,16 @@ function Execute {
     Param([String]$File)
 
     try {
-        $R = (Start-Process -FilePath $File -Wait -NoNewWindow -PassThru).ExitCode
-        $Code = if (Test-Path $File) {$R} Else {127}
-        return $Code
+        $process = Start-Process -FilePath $File -NoNewWindow -PassThru
+        $Handle = $process.Handle
+        $process.WaitForExit()
+        $exitCode = $process.ExitCode
+
+        if (-not (Test-Path $File)) {
```

Comment:
nit: prefer positive conditions instead of negative unless there's a good reason (`if Test-Path` rather than `if not Test-Path`)